lib/image-stream/gzip.js

```diff
@@ -38,37 +35,34 @@ const ISIZE_LENGTH = 4;
  * by reading its `ISIZE`. The specification clarifies that this
  * value is just an estimation.
```

Comment:
I wonder if it's worth adding a comment here that this involves reading from the very end of the file? If we're streaming, we wouldn't want to download the whole file twice ;-)